engines/dfc_provider/app/controllers/dfc_provider/enterprise_groups/affiliated_by_controller.rb

```diff
@@ -22,6 +22,14 @@ def create
         head :created
       end
 
+      def destroy
+        group = EnterpriseGroup.find(params[:enterprise_group_id])
+
+        authorize! :update, group
+
+        group.enterprises.delete(params[:id])
```

Comment:
Is there no equivalent for `head :created`?

Reply:
There is no `:deleted` but we could use `:accepted` (status 202) instead of `:no_content` (status 204), what do you think ?

Reply:
I just looked around a bit and `:no_content` seems to be the best option. `:accepted` would imply that the resource is scheduled for deletion but still exists which is not true.